<!doctype html>
<html lang="en">
<head>
  <title>Assays | MCF10A Dense Data Cube</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>

  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.css">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.3.15/slick.css">
  <link rel="stylesheet" href="https://cdn.datatables.net/1.10.12/css/jquery.dataTables.min.css">
  <link rel="stylesheet" href="{{ url_for('static', filename='css/style.css') }}">
</head>
<body>
  {% include "partials/header.html"%}
  <div class="container">{% block content %}{% endblock %}
  <h1 style="margin-top: 0px;font-family: monospace;">File Format</h1>
  <h2>GR</h2>
  <p><a href="http://eh3.uc.edu/apps/grtutorial/Home.html" target="_blank">GR metrics</a> are derived from normalized growth rate inhibition (GR) values which are based on the ratio of growth rates in the presence and absence of perturbagen. Largely independent of cell division rate and assay duration, GR metrics are more robust than IC<sub>50</sub> and E<sub>max</sub> for assessing cellular response to drugs, RNAi, and other perturbations in which control cells divide over the course of the assay.</p>
  <h3 id="cellcounts">Cell Counts</h3>
  <p>Columns of the file include:
      <ul>
        <li>cell_line</li>
        <li>agent</li>
        <li>concentration</li>
        <li>GRvalue</li>
        <li>cell_count</li>
        <li>cell_count__ctrl</li>
        <li>cell_count__time0</li>
        <li>Replicate</li>
      </ul>
  </p>
  <h3 id="grmetrics">GR Metrics</h3>
  <p>Columns of the file include:
      <ul>
        <li>cell_line</li>
        <li>agent</li>
        <li>time</li>
        <li>GR50</li>
        <li>GRmax</li>
        <li>GR_AOC</li>
        <li>GEC50</li>
        <li>GRinf</li>
        <li>Hill</li>
        <li>Replicate</li>
      </ul>
  </p>
  <h2 id="l1000">L1000</h2>
  <p><a href="http://www.lincscloud.org/l1000/" target="_blank">L1000</a> is a gene expression profiling assay based on the direct measurement of a reduced representation of the transcriptome and computational inference of the portion of the transcriptome not explicitly measured.</p>
  <h3>GCT</h3>
  <p>GCT is a matrix tab delimited file format. In addition to a matrix of numerical data a gct file contains row and column ids and annotations. <a href="http://support.lincscloud.org/hc/en-us/articles/202105453-GCT-Gene-Cluster-Text-Format-" target="_blank">Learn more</a></p>
  <h2 id="p100">P100</h2>
  <p><a href="http://www.broadinstitute.org/~jjaffe/p100.php" target="_blank">P100</a> assay consists of a set of 96 phosphopeptide probes that are easily observable and quantifiable in diverse cell types ranging from embryonic stem cells to human leukemia cells, and that they behave in interesting and orthogonal ways in response to varied chemical and genetic pertrubations. </p>
  <h3>GCT</h3>
  <p>GCT is a matrix tab delimited file format. In addition to a matrix of numerical data a gct file contains row and column ids and annotations. <a href="http://support.lincscloud.org/hc/en-us/articles/202105453-GCT-Gene-Cluster-Text-Format-" target="_blank">Learn more</a></p>     
  <p>Row headers include:
      <ul>
        <li>pr_gene_id</li>
        <li>pr_gene_symbol</li>
        <li>pr_p100_base_peptide</li>
        <li>pr_p100_cluster</li>
        <li>pr_p100_gene_cluster_code</li>
        <li>pr_p100_modified_peptide_code</li>
        <li>pr_p100_original_probe_id</li>
        <li>pr_p100_phosphosite</li>
        <li>pr_probe_normalization_group</li>
        <li>pr_probe_suitability_manual</li>
        <li>pr_uniprot_id</li>
      </ul>
  </p>
  <p>Column headers include:
      <ul>
        <li>cell_id</li>
        <li>det_filename</li>
        <li>det_normalization_group_vector</li>
        <li>det_plate</li>
        <li>det_well</li>
        <li>isomeric_smiles</li>
        <li>lsm_id</li>
        <li>pert_batch_internal_compound_enumerator</li>
        <li>pert_batch_internal_replicate</li>
        <li>pert_dose</li>
        <li>pert_dose_unit</li>
        <li>pert_id</li>
        <li>pert_iname</li>
        <li>pert_time</li>
        <li>pert_time_unit</li>
        <li>pert_type</li>
        <li>pert_vehicle</li>
        <li>provenance_code</li>
        <li>pubchem_cid</li>
      </ul>
  </p>

  <h2 id="gcp">Global Chromatin Profiling (GCP)</h2>
  <p><a href="http://www.broadinstitute.org/~jjaffe/p100.php" target="_blank">Global chromatin profiling (GCP)</a> technology is a mass spectrometry-based assay to identify and quantify post-translational modifications on histone proteins from bulk chromatin.</p>
  <h3>GCT</h3>
  <p>GCT is a matrix tab delimited file format. In addition to a matrix of numerical data a gct file contains row and column ids and annotations. <a href="http://support.lincscloud.org/hc/en-us/articles/202105453-GCT-Gene-Cluster-Text-Format-" target="_blank">Learn more</a></p>
  <p>Row headers include:
      <ul>
        <li>pr_gcp_base_peptide</li>
        <li>pr_gcp_BI_number</li>
        <li>pr_gcp_histone_mark</li>
        <li>pr_gcp_modified_peptide_code</li>
        <li>pr_gene_id</li>
        <li>pr_gene_symbol</li>
        <li>pr_probe_normalization_group</li>
        <li>pr_probe_suitability_manual</li>
        <li>pr_uniprot_id</li>
      </ul>
  </p>
  <p>Column headers include:
      <ul>
        <li>cell_id</li>
        <li>cell_reprogrammed</li>
        <li>det_filename</li>
        <li>det_normalization_group_vector</li>
        <li>det_plate</li>
        <li>det_well</li>
        <li>isomeric_smiles</li>
        <li>lsm_id</li>
        <li>pert_batch_internal_compound_enumerator</li>
        <li>pert_batch_internal_replicate</li>
        <li>pert_dose</li>
        <li>pert_dose_unit</li>
        <li>pert_id</li>
        <li>pert_iname</li>
        <li>pert_time</li>
        <li>pert_time_unit</li>
        <li>pert_type</li>
        <li>pert_vehicle</li>
        <li>provenance_code</li>
        <li>pubchem_cid</li>
      </ul>
  </p>

  <h2 id="cycif">Cyclic Immunofluorescence (CycIF)</h2>
  <p><a href="http://lincs.hms.harvard.edu/db/datasets/20303/main" target="_blank">Cyclic immunofluorescence (CycIF)</a> is a fluorescence imaging assay that measures protein levels and phosphorylation states in cells.</p>
  <h3>CycIF Mean ZScores</h3>
  <p>File contains matrix of mean zscores of immunofluorescent intensity measurements for each combination of agent, concentration, time, and compartment (cytosol or nucleus). Zscoring implemented for each combination using the mean and standard deviation of that combination's compartment.</p>
  <p>Columns of the file include:
    <ul>
      <li>agent</li>
      <li>concentration</li>
      <li>time</li>
      <li>compartment</li>
    </ul>
  </p>
  <h3>CycIF Raw</h3>
  <p>Data file contains matrix of immunofluorescent intensity measurements for cells treated with different doses of a perturbagen. See datacolumn_LUT Excel file in downloaded dataset results directory from <a href="http://lincs.hms.harvard.edu/db/datasets/20303/results" target="_blank">HMS LINCS</a> for additional column header details.</p>

    <h2 id="mwa">Microwestern Arrays (MWA)</h2>
  <p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2881471/" target="_blank">Microwestern Arrays (MWA)</a> is a high-throughput variation of the Western blot, which uses electrophoresis to determine protein levels and modifications by measuring fluorescence intensity.</p>
  <h3>MWA Measurements</h3>
  <p>File contains matrices of fluorescence intensity measurements of a target protein and alpha tubulin for cells treated with different doses of a perturbagen, as well as the ratios between target protein and alpha tubulin for each perturbation.</p>
  <p>Columns of the file include:
    <ul>
      <li>Ave</li>
      <li>S.D.</li>
      <li>C.V.</li>
      <li>ss</li>
    </ul>
  </p>

    <h2 id="mema">Microenvironment Microarray (MEMA)</h2>
  <p><a href="https://www.synapse.org/#!Synapse:syn2862345/wiki/232002" target="_blank">Microenvironment Microarray (MEMA)</a> is a measures cellular response to almost 3000 different microenvironment perturbagens (MEPs).</p>
  <h3>SS1 Level4, SS2 Level4, SS3 Level4, SSC Level4</h3>
  <p>File contains matrix of normalized measurements for different cellular characteristics after perturbation.</p>
  <p>Columns of the file include:
    <ul>
      <li>MEP_Drug</li>
      <li>Barcode</li>
      <li>Ligand</li>
      <li>ECMp</li>
      <li>CellLine</li>
      <li>PinDiameter</li>
      <li>Ligand1</li>
      <li>Ligand2</li>
      <li>ECM1</li>
      <li>ECM2</li>
      <li>CellLineAn</li>
      <li>CellLineLx</li>
      <li>Ligand1Lx</li>
      <li>ECM1Lx</li>
      <li>ECM2Lx</li>
      <li>ECM3Lx</li>
      <li>MEP</li>
      <li>k</li>
      <li>NormMethod</li>
    </ul>
  </p>
</div>


  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
</body>
</html>