{% extends "drug-wrapper.html" %}
{% block content %}
    <div class="drug-dropdown">
      <select class="all-drugs" id="drug-dropdown">
        <option {{'selected="selected"' if drug_selected == "trametinib" else "" }} href="trametinib">Trametinib</option>
        <option {{'selected="selected"' if drug_selected == "alpelisib" else "" }} href="alpelisib">Alpelisib</option>
        <option {{'selected="selected"' if drug_selected == "vorinostat" else "" }} href="vorinostat">Vorinostat</option>
        <option {{'selected="selected"' if drug_selected == "palbociclib" else "" }} href="palbociclib">Palbociclib</option>
        <option {{'selected="selected"' if drug_selected == "neratinib" else "" }} href="neratinib">Neratinib</option>
        <option {{'selected="selected"' if drug_selected == "paclitaxel" else "" }} href="paclitaxel">Paclitaxel</option>
        <option {{'selected="selected"' if drug_selected == "etoposide" else "" }} href="etoposide">Etoposide</option>
        <option {{'selected="selected"' if drug_selected == "dasatinib" else "" }} href="dasatinib">Dasatinib</option>
      </select>
    </div>

    <div class="drug-subheading">
      <p>{{ drug_info['kinase'] }}</p>
      <p>{{ drug_info['brd'] }}</p>
    </div>
    <div class="navigation-container" style="font-weight:bold; font-size:1.1em;">
      <a href="#GR">GR Metrics</a>
      {% if 'L1000' in drug_info['assays'] %}
        |<a href="#Enrichr">GR + L1000 (Enrichr)</a>
        |<a href="#L1000">L1000</a>
        |<a href="#L1000CDS2">L1000CDS2</a>
      {% endif %}
      {% if 'P100' in drug_info['assays'] %}
        |<a href="#P100">P100</a>
      {% endif %}
      {% if 'GCP' in drug_info['assays'] %}
        |<a href="#GCP">GCP</a>
      {% endif %}
      {% if 'CycIF' in drug_info['assays'] %}
        |<a href="#CycIF">CycIF</a>
      {% endif %}
    </div>

    <div class="description">
      {{drug_info['description']|safe}}
      <div class="pubchem-link">Learn more on <a href="{{ drug_info['pubchem'] }}" target="_blank"><img class="pubchem-logo" src="{{ url_for('static', filename='image/pubchem-logo.svg') }}"></a></div>
      <br>
      More drug information can be found on 
      <table style="width:60%;text-align:center; margin: 0 auto;">
        <tr>
          <td><a class="lincs-logo" href="{{ drug_info['lincs'] }}" target="_blank">LINCS Data Portal</a></td>
          {% if drug_info['drugbank'] != "" %}
            <td><a href="{{ drug_info['drugbank'] }}" target="_blank"><img class="pubchem-logo" src="{{ url_for('static', filename='image/drugbank-logo.png') }}"></a></td>
          {% endif %}
          {% if drug_info['chembl'] != "" %}
            <td><a class="chembl-logo" href="{{ drug_info['chembl'] }}" target="_blank">ChEMBL</a></td>
          {% endif %}
        </tr>
      </table>
    </div>

    <script>
      window.data_conc = {{data_conc | tojson}};
      window.drug = "{{drug_selected}}";
      window.assayColor = {{assay_color | tojson}};
    </script>

    <div class="data-overview"> 
      <h2>Data Available</h2>
      <table style="width:60%; /*text-align:center;*/ margin: 0 auto; ">
        <tr>
          <td style="vertical-align: top;" rowspan="{{ data_available.time|length * 2 + 1}}">
            <div class="data-box-title" >
              GR Metrics
            </div>
            <div id="gr-curve-container">
              
            </div>
          </td>
          <td class="data-box-title">
            Time
          </td>
          <td class="data-box-title">
            Assay
          </td>
          <td class="data-box-title">
            Concentration (&mu;M)
          </td>
        </tr>

        {% for i in range(data_available.time|length) %}
          <tr class="data-available">
            {% if data_available.time[i][0] != "" %}
              <td rowspan="{{ data_available.time[i][1]}}" onmouseover="displayConcentrations('time', this);" style="text-align:center; padding: 20px 0px 20px 0px;">
                {% set previousTime = data_available.time[i][0] %}
                {{ data_available.time[i][0] }}
              </td>
            {% endif %}

            <td style="padding-left: 5px; color: {{assay_color[data_available.assay[i]]}}" onmouseover="displayConcentrations('assay-{{ previousTime }}', this);">
              {{ data_available.assay[i] }}
            </td>
            <td style="padding-left: 5px;" onmouseover="displayConcentrations('conc-{{ data_available.assay[i] }}', this);">
              {% for conc in data_available.concentration[i]%}
                {{ conc }} &emsp;
              {% endfor %}
            </td>

          </tr>
        {% endfor %}

        </table>

          </div> <!-- /.data-overview -->
          <div class="assay-content">
            <h2 id="GR">GR Metrics</h2>
            <div class="description">
              <p>
                <a href="http://eh3.uc.edu/apps/grtutorial/Home.html" target="_blank">GR metrics</a> are derived from normalized growth rate inhibition (GR) values which are based on the ratio of growth rates in the presence and absence of perturbagen. Largely independent of cell division rate and assay duration, GR metrics are more robust than IC<sub>50</sub> and E<sub>max</sub> for assessing cellular response to drugs, RNAi, and other perturbations in which control cells divide over the course of the assay.
              </p>
          
          <div class="gr-equation">
                <p>Each GR curve is fitted to a sigmoidal curve as follows</p>
                <math>
                  <!-- <mrow> -->
                  <mi>GR(c)</mi>
                  <mo>=</mo>
                  <msub>
                    <mi>GR</mi>
                    <mn>inf</mn>
                  </msub>
                  <mo>+</mo>
                  <mfrac>
                    <mrow>
                      <mi>1</mi>
                      <mo>&#x2212;</mo>
                      <msub>
                        <mi>GR</mi>
                        <mn>inf</mn>
                      </msub>
                    </mrow>
                    <mrow>
                      <mi>1</mi>
                      <mo>&#x2b;</mo>
                      <mo>(</mo>
                      <mi>c</mi>
                      <mo>/</mo>
                      <msub>
                        <mi>GEC</mi>
                        <mn>50</mn>
                      </msub>
                      <msup>
                        <mo>)</mo>
                        <mn>
                          <msub>
                          <mi>h</mi>
                          <mn>GR</mn>
                          </msub>
                        </mn>
                      </msup>
                    </mrow>
                  </mfrac>
                  <!-- </mrow> -->
                </math>
              </div>
            </div> <!-- /.gr-equation -->

            <div class="gr-curve"></div>
            <div class="gr-equation">
              where the fitted parameters are:
              <br>
              <ul>
                <li>GEC<sub>50</sub> = concentration at half-maximal effect</li>
                <li>GR<sub>Inf</sub> = effect of drug at infinite concentration</li>
                <li>h<sub>GR</sub> = (Hill coefficient) reflects how steep the dose-response curve is</li>
              </ul>
            </div>

            <!-- <div class="gr50-boxplot"></div> -->
            <div class="download-table">
              <a href="../static/data/gr/metrics/MCF10A_{{drug_selected}}_GR.tsv">download GR metrics</a>
            </div>
          </div> <!-- /.assay-content -->

          {% if 'L1000' in drug_info['assays'] %}
            <div class="assay-content">
              <h2 id="Enrichr">GR + L1000 (Enrichr)</h2>
              <div class="description">
                <p>
                  <a href="http://amp.pharm.mssm.edu/Enrichr/" target="_blank">
                    Enrichr</a> is a web tool that performs gene set enrichment analysis. The value of each cell represents the <b>combined score</b>, a combination of the p-value and z-score. Use the dropdown to change Enrichr's enrichment library. 
                  </p>
                  {% with assay="Enrichr", drug=drug_selected, time=['3h', '24h'], default_time="24h", default_library="ChEA_2015"%}
                    {% include "partials/drug-enrichr.html" %}
                  {% endwith %}
              </div> <!-- /.assay-content -->

              <div class="assay-content">
                <h2 id="L1000">L1000</h2>
                <div class="description">
                  <a href="http://www.lincscloud.org/l1000/" target="_blank">L1000</a> is a gene expression profiling assay based on the direct measurement of a reduced representation of the transcriptome and computational inference of the portion of the transcriptome not explicitly measured. The value of each cell represents the <b>differential expression</b> of each gene determined by the Characteristic Direction method. 
                </div>
                {% with assay="L1000", drug=drug_selected, time=['3h', '24h'], default_time="3h", default_regulation="down" %}
                  {% include "partials/drug-l1000.html" %}
                {% endwith %}
              </div> <!-- /.assay-content (L1000) -->

              <div class="assay-content">
                <h2 id="L1000CDS2">L1000CDS2</h2>
                <div class="description">
                  <a href="http://amp.pharm.mssm.edu/L1000CDS2" target="_blank">L1000CDS2</a> is a web tool that queries the LINCS L1000 dataset to identify small molecules that can reverse or mimic the observed input expression pattern.
                </div>
                {% with assay="L1000CDS2", time=['3h', '24h'], default_time="3h" %}
                  {% include "partials/drug-l1000cds2.html" %}
                {% endwith %}
              </div> <!-- /.assay-content (L1000CDS2) -->
      {% endif %}

  {% if 'P100' in drug_info['assays'] %}
    <div class="assay-content">
      <h2 id="P100">P100</h2>
      <div class="description">
        <a href="http://www.broadinstitute.org/~jjaffe/p100.php" target="_blank">P100</a> assay consists of a set of 96 phosphopeptide probes that are easily observable and quantifiable in diverse cell types ranging from embryonic stem cells to human leukemia cells, and that they behave in interesting and orthogonal ways in response to varied chemical and genetic pertrubations. The value of each cell represents the <b>z score</b>.
      </div>

      {% with assay="P100", drug=drug_selected, time=['3h'], default_time="3h"%}
          {% include "partials/drugs-clustergrammer.html" %}
      {% endwith %}
    </div> <!-- /.assay-content (P100) -->
  {% endif %}

  {% if 'GCP' in drug_info['assays'] %}
    <div class="assay-content">
      <h2 id="GCP">GCP</h2>
      <div class="description">
        <a href="http://www.broadinstitute.org/~jjaffe/p100.php" target="_blank">Global chromatin profiling (GCP)</a> technology is a mass spectrometry-based assay to identify and quantify post-translational modifications on histone proteins from bulk chromatin. The value of each cell represents the <b>z score</b>.
      </div>
      </div>
      {% with assay="GCP", drug=drug_selected, time=['24h'], default_time="24h"%}
          {% include "partials/drugs-clustergrammer.html" %}
      {% endwith %}
    </div> <!-- /.assay-content (GCP) -->
  {% endif %}

  {% if 'GCP' in drug_info['assays'] %}
    <div class="assay-content">
      <h2 id="GCP">GCP</h2>
      <div class="description">
        <a href="http://www.broadinstitute.org/~jjaffe/p100.php" target="_blank">Global chromatin profiling (GCP)</a> technology is a mass spectrometry-based assay to identify and quantify post-translational modifications on histone proteins from bulk chromatin. The value of each cell represents the <b>z score</b>.
      </div>
      </div>
      {% with assay="GCP", drug=drug_selected, time=['48h'], default_time="48h"%}
          {% include "partials/drugs-clustergrammer.html" %}
      {% endwith %}
    </div> <!-- /.assay-content (GCP) -->
  {% endif %}


{% endblock %}